docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  app:
+    build: .
+    ports:
+      - "${SERVER_PORT}:3000"
+    depends_on:
+      - db
+    environment:
+      - DATABASE_URL=postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}
+      - JWT_SECRET=${JWT_SECRET}
+  db:
+    image: postgres:13
+    environment:
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_DB=${DB_NAME}
```

Comment:
sorrry i didn't get you. sorry

Reply:
Usually, we add an extra line at the end of the file. You can see a red minus icon in your changes; that's because it is not there in your files.

So just add an extra line at the end like the suggestion above @mayura-andrew

Reply:
You can read more about it here: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log